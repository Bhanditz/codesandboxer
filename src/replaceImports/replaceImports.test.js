import cases from 'jest-in-case';
import replaceImports from './';

cases(
  'replaceImports()',
  ({ files, replaces }) => {
    let code = replaceImports(files, replaces);
    expect(code).toMatchSnapshot();
  },
  [
    {
      name: 'replace imports',
      files: `import a from 'b' import c from 'd'`,
      replaces: [['b', 'z'], ['d', 'arg']],
    },
  ],
);
