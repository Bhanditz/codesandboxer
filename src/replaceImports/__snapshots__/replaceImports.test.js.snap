// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`replaceImports() replace imports 1`] = `"import a from 'z' import c from 'arg'"`;
